analyzers/PDebugger.py

```diff
@@ -0,0 +1,30 @@
+from heppy.framework.analyzer import Analyzer
+import logging
+import heppy.utils.pdebug as pdebugging
```

Comment:
please import as pdebug instead of pdebugging (changing module names at import can be misleading and should be avoided if possible)